### PR TITLE
Migrate to DockerHub

### LINKED ISSUE #91: Migrate to DockerHub

### PR DESCRIPTION
**Pull request by neongreen**
Fixes #91.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -90,19 +87,19 @@ jobs:
...
         # Tag the image with commit hash and push
         - HASH=$(git log -1 --pretty=%h)
-        - docker tag juspay/fencer "docker.pkg.github.com/juspay/fencer/fencer:$HASH"
-        - docker push "docker.pkg.github.com/juspay/fencer/fencer:$HASH"
+        - docker tag juspayin/fencer "juspayin/fencer:$HASH"
+        - docker push "juspayin/fencer:$HASH"
         # Tag the same image with "master" and push
-        - docker tag juspay/fencer docker.pkg.github.com/juspay/fencer/fencer:master
-        - docker push docker.pkg.github.com/juspay/fencer/fencer:master
-      if: branch = master AND type = push
```
Comment (mdimjasevic) [inline .travis.yml:108]: why is the check `branch = master` not needed anymore?

File: .travis.yml [hunk 2]
```diff
@@ -67,7 +64,7 @@ jobs:
             --name fencer-server
             --network test
             --detach
-            juspay/fencer
+            juspayin/fencer
```
Comment (mdimjasevic) [inline .travis.yml]: Even though it is not a big deal, I'd suggest using an environment variable instead of a string literal.

**Review by mdimjasevic — APPROVED** [review]
Overall this is good. I made a couple of tiny remarks, which are inlined.

File: .travis.yml (hunk 1 above)
Comment (neongreen) [reply .travis.yml:108]: It's a part of the `TEST COMMIT WILL REVERT LATER` commit. I'll revert it once it becomes evident that the PR works.

**Comment by neongreen** [comment]
@mdimjasevic okay, it works (https://hub.docker.com/r/juspayin/fencer/tags) and can be rereviewed. Not sure whether my use of environment variables is okay or not quite.

**Review by mdimjasevic — APPROVED** [review]

**Comment by mdimjasevic** [comment]
> @mdimjasevic okay, it works (https://hub.docker.com/r/juspayin/fencer/tags) and can be rereviewed. Not sure whether my use of environment variables is okay or not quite.

It's great to see this working!

Regarding environment variables: in my view it's easier to understand when suggestive environment variable names are used instead of string literals, but this is just to aid understanding what those string literals represent. I like the change you did in that regard.

**Pull request merged**